gui.py

```
@@ -1,487 +1,491 @@
+        ttk.Button(actions, text="Draw Chladni pattern", command=self.draw_pattern).pack(side="left", padx=4)
+        ttk.Button(actions, text="Play isochronic preview", command=self.preview_visualization).pack(side="left", padx=4)
+        self.pattern_host = ttk.Frame(tab)
+        self.pattern_host.pack(fill="both", expand=True)
+
+    def _run_background(self, working_message: str, job):
+        self.status.set(working_message)
+
+        def worker():
+            try:
+                result = job()
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)